python/paddle/fluid/tests/unittests/dygraph_to_static/test_mnist.py

```diff
@@ -156,15 +160,23 @@ def test_mnist_declarative(self):
             msg='dygraph is {}\n static_res is \n{}'.format(dygraph_loss,
                                                             static_loss))
 
-    def train(self, to_static=False):
+    def test_mnist_declarative_cpu_vs_mkldnn(self):
```

Comment:
Where is this used?

Reply:
This is just a unit test. I added a new test case comparing the mkldnn version of mnist with non-mkldnn.